ipsec_accel/common/utils.py

```diff
@@ -0,0 +1,315 @@
+#!/usr/bin/python
```

Comment:
This is duplicated code from ipu_playbook/ovs_offload/common/utils.py

We should remove this from here and re-use the code used in the previously committed location. I see some new code in functions `split_mac`, `split_mac_2` and `ip_dec_to_hex`. These can be added to the other file to bring it in line with the needed functionality